src/cmiss_petsc.f90

```diff
@@ -5705,18 +5768,19 @@ END SUBROUTINE PETSC_TSSETPROBLEMTYPE
   !
     
   !>Buffer routine to the PETSc TSSetRHSFunction routine.
-  SUBROUTINE PETSC_TSSETRHSFUNCTION(TS_,RHSFUNCTION,CTX,ERR,ERROR,*)
+  SUBROUTINE PETSC_TSSETRHSFUNCTION(TS_,PETSC_RATES,RHSFUNCTION,CTX,ERR,ERROR,*)
 
     TYPE(PETSC_TS_TYPE), INTENT(INOUT) :: TS_ !<The TS to set the problem type for
+    TYPE(PETSC_VEC_TYPE), INTENT(INOUT) :: PETSC_RATES
     EXTERNAL RHSFUNCTION !<The external RHS function to call
-    TYPE(SOLVER_TYPE), POINTER :: CTX !<The solver data to pass to the function
+    TYPE(CELLML_PETSC_CONTEXT_TYPE), POINTER :: CTX !<The solver data to pass to the function
```

Comment:
I'm not sure why solver_type has been changed to cellml_petsc_context_type here. Solver it the generic here and everything should be accessed through it?